test/e2e_node/security_context_test.go

```diff
@@ -114,4 +115,75 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			}
 		})
 	})
+
+	Context("when creating a pod in the host network namespace", func() {
+		makeHostNetworkPod := func(podName, image string, command []string, hostNetwork bool) *v1.Pod {
+			return &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: podName,
+				},
+				Spec: v1.PodSpec{
+					RestartPolicy: v1.RestartPolicyNever,
+					HostNetwork:   hostNetwork,
+					Containers: []v1.Container{
+						{
+							Image:   image,
+							Name:    podName,
+							Command: command,
+						},
+					},
+				},
+			}
+		}
+		listNetworkInterfaceCommand := []string{"sh", "-c", "cat /proc/net/dev | awk '{print $1}'"}
```

Comment:
`cat /proc/net/dev | awk '{print $1}'` can be simplified into `awk '{print $1}' /proc/net/dev`